x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/ner/ner_output.tsx

```diff
@@ -0,0 +1,146 @@
+  const lineSplit: JSX.Element[] = [];
+  result.forEach(({ value, entity }) => {
+    if (entity === null) {
+      const lines = value
+        .split(/(\n)/)
+        .map((line) => (line === '\n' ? <br /> : <span>{line}</span>));
+
+      lineSplit.push(...lines);
+    } else {
+      lineSplit.push(
+        <EuiToolTip
+          position="top"
+          content={
+            <div>
+              <div>
+                <EuiIcon
+                  size="m"
+                  type={getClassIcon(entity.class_name)}
+                  style={{ marginRight: ICON_PADDING, verticalAlign: 'text-top' }}
+                />
+                {value}
+              </div>
+              <EuiHorizontalRule margin="none" />
+              <div style={{ fontSize: euiTheme.euiFontSizeXS, marginTop: ICON_PADDING }}>
+                <div>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.testModelsFlyout.ner.output.typeTitle"
+                    defaultMessage="Type"
+                  />
+                  : {getClassLabel(entity.class_name)}
+                </div>
+                <div>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.testModelsFlyout.ner.output.probabilityTitle"
+                    defaultMessage="Probability"
+                  />
+                  : {entity.class_probability}
+                </div>
+              </div>
+            </div>
+          }
+        >
+          <EntityBadge entity={entity}>{value}</EntityBadge>
+        </EuiToolTip>
+      );
+    }
+  });
+  return <div style={{ lineHeight: '24px' }}>{lineSplit}</div>;
+};
+
+const EntityBadge = ({
+  entity,
+  children,
+}: {
+  entity: estypes.MlTrainedModelEntities;
+  children: ReactNode;
+}) => {
+  const { euiTheme } = useCurrentEuiTheme();
+  return (
+    <EuiBadge
+      color={getClassColor(euiTheme, entity.class_name)}
+      style={{
+        marginRight: ICON_PADDING,
+        marginTop: `-${ICON_PADDING}`,
+        border: `1px solid ${getClassColor(euiTheme, entity.class_name, true)}`,
+        fontSize: euiTheme.euiFontSizeXS,
+        padding: '0px 6px',
+      }}
+    >
+      <EuiFlexGroup gutterSize="none">
+        <EuiFlexItem grow={false}>
+          <EuiIcon
+            size="s"
+            type={getClassIcon(entity.class_name)}
+            style={{ marginRight: ICON_PADDING, marginTop: ICON_PADDING }}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>{children}</EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiBadge>
+  );
+};
+
+function getClassIcon(className: string) {
+  switch (className) {
+    case 'PER':
+      return 'user';
+    case 'LOC':
+      return 'visMapCoordinate';
+
+    default:
+      return 'cross';
+  }
+}
+
+function getClassLabel(className: string) {
+  switch (className) {
+    case 'PER':
+      return 'Person';
+    case 'LOC':
+      return 'Location';
+
+    default:
+      return 'cross';
```

Comment:
Is there another term that can be displayed for the `default` case, like `other`?

![image](https://user-images.githubusercontent.com/7405507/160449889-3e08d608-8252-4501-bb1e-c1afab743376.png)

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/a012a58b44ebcc7b1f3b9c70fc375cb3bf7a2ede

`MISC` entities are now show like this:
![image](https://user-images.githubusercontent.com/22172091/160596890-85de2efa-b044-4c18-9b6a-839f9308c2f7.png)

Unknown entries will have the type `Unknown` and the same `?` icon